frontend/src/store/experiments.js

```diff
@@ -123,6 +123,7 @@ export default {
       if (Object.keys(state.units).length === 0) {
         this._vm.$defaultApi.getUnits().then(
           (data) => commit("storeUnits", data),
+          // eslint-disable-next-line no-console
```

Comment:
If you want only to allow console.error globally you could always configure eslintrc for that https://eslint.org/docs/latest/rules/no-console